Tests/StreamChatUITests/SnapshotTests/ChatThread/ChatThreadVC_Tests.swift

```diff
@@ -78,6 +78,34 @@ final class ChatThreadVC_Tests: XCTestCase {
         )
     }
 
+    func test_whenShouldMessagesStartAtTheTopEnabled() {
+        var components = Components.mock
+        components.shouldMessagesStartAtTheTop = true
+        vc.components = components
+
+        channelControllerMock.simulateInitial(
+            channel: .mock(cid: .unique),
+            messages: [],
+            state: .remoteDataFetched
+        )
+        messageControllerMock.simulateInitial(
+            message: .mock(id: .unique, cid: .unique, text: "First message", author: .mock(id: .unique), replyCount: 2),
+            replies: [
+                .mock(id: .unique, cid: .unique, text: "First reply", author: .mock(id: .unique)),
+                .mock(id: .unique, cid: .unique, text: "Second reply", author: .mock(id: .unique))
+            ],
+            state: .localDataFetched
+        )
+        messageControllerMock.simulate(state: .remoteDataFetched)
+
+        vc.view.layoutIfNeeded()
+
+        AssertSnapshot(
+            vc,
+            variants: [.smallDark]
```

Comment:
Same as before, probably we can use at least `.smallLight` too

Reply:
We don't have a smallLight for now

Reply:
Can you elaborate why we can't use the rest of the variants?